LibAddonMenu-2.0/LibAddonMenu-2.0/controls/iconpicker.lua

```diff
@@ -134,9 +134,12 @@ function IconPickerMenu:Initialize(name)
 end
 
 function IconPickerMenu:OnMouseEnter(icon)
-    InitializeTooltip(InformationTooltip, icon, TOPLEFT, 0, 0, BOTTOMRIGHT)
-    SetTooltipText(InformationTooltip, LAM.util.GetStringFromValue(icon.tooltip))
-    InformationTooltipTopLevel:BringWindowToTop()
+    local tooltipText = icon.tooltip and LAM.util.GetStringFromValue(icon.tooltip)
+    if tooltipText and tooltipText ~= "" then
+        InitializeTooltip(InformationTooltip, icon, TOPLEFT, 0, 0, BOTTOMRIGHT)
+        SetTooltipText(InformationTooltip, text)

```

Comment:
Shouldn't you pass `tooltipText`? `text` is not defined ;)
Other than that it looks good now.

Reply:
fixed, thx for the hint